query/README.md

```diff
@@ -0,0 +1,146 @@
+Keen Query Clients
+===================
+
+[![Build Status](https://travis-ci.org/keenlabs/KeenClient-Java.png?branch=master)](https://travis-ci.org/keenlabs/KeenClient-Java)
```

Comment:
Nit: Probably not much point in leaving this here, since it's in the repo-global README file.